docs/pages/versions/unversioned/sdk/ui.mdx

```diff
@@ -0,0 +1,206 @@
+---
+title: UI
+description: A set of platform components to build native-feeling UIs.
+sourceCodeUrl: 'https://github.com/expo/expo/tree/main/packages/expo-ui'
+packageName: '@expo/ui'
+platforms: ['android', 'ios']
+---
+
+import APISection from '~/components/plugins/APISection';
+import { APIInstallSection } from '~/components/plugins/InstallSection';
+import { ContentSpotlight } from '~/ui/components/ContentSpotlight';
+import { Tabs, Tab } from '~/ui/components/Tabs';
+
+`@expo/ui` is a set of native input components that allows you to build native-feeling forms.
+
+> **warning** This library is currently in alpha and subject to breaking changes. It is not available in the Expo Go app.
+
+## Installation
+
+<APIInstallSection />
+
+## Examples
+
+### Button
+
+<Tabs>
+<Tab label="iOS Screenshot">
+
+<ContentSpotlight
+  alt="Button component on iOS."
+  src="/static/images/expo-ui/button/ios.png"
+  className="max-w-[480px]"
+/>
+
+</Tab>
+
+<Tab label="Android Screenshot">
```

Comment:
Can we make Android tab the initial tab for all sections where there is an Android example? Relates to the platform order from our writing style guide: https://github.com/expo/expo/blob/main/guides/Expo%20Documentation%20Writing%20Style%20Guide.md#platform-order-when-referencing-android-ios-and-web-or-multiple-platforms